env.c

```diff
@@ -48,8 +48,11 @@ char *RETRACE_IMPLEMENTATION(getenv)(const char *envname)
 {
 	rtr_getenv_t real_getenv = RETRACE_GET_REAL(getenv);
 	char *env = real_getenv(envname);
-	trace_printf(1, "getenv(\"%s\"); [%s]\n", envname, env);
-	return real_getenv(envname);
+	if (env != NULL)
+	    trace_printf(1, "getenv(\"%s\"); [\"%s\"]\n", envname, env);
+	else
+	    trace_printf(1, "getenv(\"%s\"); [NULL]\n", envname, env);
+	return (env);
```

Comment:
That's correct, I still confuse Linux and BSD styles sometimes.  Good thing checkpatch.pl isn't confused by it.